libpod/container_internal_common.go

```diff
@@ -177,6 +179,20 @@ func getOverlayUpperAndWorkDir(options []string) (string, string, error) {
 	return upperDir, workDir, nil
 }
 
+var hasCapSysResourceOnce = sync.OnceValues(hasCapSysResource)
+
+// hasCapSysResource returns whether the current process has CAP_SYS_RESOURCE.
+func hasCapSysResource() (bool, error) {
```

Comment:
if you define this function directly in the OnceValues() call you do not have to give it a name in the global scope
`sync.OnceValues(func() (bool, error) {...})`

Reason being is that this way it is impossible for anyone to call it without the cache accidentally and you can name it hasCapSysResource instead of hasCapSysResourceOnce

Reply:
thanks, that is even better.